Code/ecg.py

```diff
@@ -0,0 +1,35 @@
+class ECG:
+
+    def __init__(self, file, window, brady_max, tachy_min):
+        from Input_csv_file import read_in
```

Comment:
hm no need to have a capital in your module name (e.g. `Input`), generally is not canonical